src/components/transactions/TxInfo/index.tsx

```diff
@@ -144,7 +167,7 @@ const TxInfo = ({ info, ...rest }: { info: TransactionInfo; omitSign?: boolean;
     return <CreationTx info={info} />
   }
 
-  if (isSwapTxInfo(info)) {
+  if (isCoWOrderTxInfo(info)) {
```

Comment:
I noticed we mix the CoW and Swaps terminology in the code (probably due to what CGW sends us?). This might shoot us in the foot once we have other types of swaps, e.g. on chains that CoW doesn't support.

Reply:
No, I went for it because we have 3 kind of orders:
swap, limit & twap.

swap is actually a limit order with price set to maket, but this is another story.

swap & limit orders have one structure
TWAP orders have a slightly different structure

When we only had swap and limit -> isSwapTxInfo was basically applying to those 2 order types. But now that I have Twaps I still want to treat this as an order and than later decide what to show and what not. That's why I went for a isCoWOrder...

Right now we are basically forwarding the data from CoWSwap. If at one point we have a different provider for a chain, then we will most probably have to come with our own format and CGW would have to convert the data to our format. But we are not there yet...

Do you have better ideas?

Reply:
I would just use `swap, limit & twap` as prefixes.

Reply:
on an API level there is no difference betwen swap and limit. Also the difference between swap,limit and twap is so small that I don't want to do:
isSwapOrder || isLimitOrder || isTwapOrder...

All those orders are basically managed through the same component at the end.